admin/class-acc_user_importer-admin.php

```diff
@@ -265,10 +265,36 @@ private function log_dual( $string ) {
 	private function proccess_user_data ( $users ) {
 		$GLOBALS['acc_logstr'] = "";		//Clear the API response log string
 
+		// Get user-configurable option values
 		$options = get_option('accUM_data');
-		$loginNameMapping = $options['accUM_login_name_mapping'];
+
```

Comment:
Here I noticed that Wordpress was giving Notices if the settings database did not contain a setting yet.  This can happen if the user never clicked "Save" in the settings.  In such case we have to use default values.  Unless I am mistaken, it seems like the database is not automatically filled with the default setting.